src/plugins/ABTestingService.js

```diff
@@ -29,6 +29,13 @@ const ABTestingService = {
     ABTesting.submit = ABTesting.submit || { index: 1 }
     ABTesting.submit.deleted = true
```

Comment:
```suggestion
```

Je pense qu'il faut supprimer ou mettre en commentaire la première et supprimer la seconde. Sans ça, je crains qu'on supprime des choses qu'on ne veux pas supprimer de Matomo car l'index 1 est partagé.

Reply:
Alors un point d'attention à avoir qui est la raison pour laquelle j'ai laissé ces 2 lignes c'est que si on les enlève la CI plante durant les tests end-to-end.

Du coup si c'est effectivement à supprimer il faut que j'investigue sur la raison de fail de la CI.

Reply:
Quelques éléments de réponse https://github.com/betagouv/aides-jeunes/blob/master/src/router.js#L345-L350

Reply:
Corrigé.